rewrite the mqttsn page (#1790)
emqtt/emq-docs-cn: en_US/gateway/mqttsn.md

## Introduction

<!--**Add an introductory section**: Begin the page briefly introducing the relevant protocol and the Gateway. This will provide context for users who are new to the concept.-->

> MQTT-SN (MQTT for Sensor Networks) is a lightweight pub/sub protocol for wireless sensor networks, the EMQX MQTT-SN Gateway allows these devices to connect and communicate with EMQX, bridging the gap between MQTT-SN and the standard MQTT protocol.

Reviewer: "These devices" seems vague. Do they refer to devices (sensors?) in wireless sensor networks? 

There should be a full stop after "wireless sensor networks".

This guide will teach you how to configure and use the MQTT-SN Gateway in EMQX.

::: tip

The MQTT-SN gateway is based on the [MQTT-SN v1.2](https://www.oasis-open.org/committees/download.php/66091/MQTT-SN_spec_v1.2.pdf).

:::

<!--a brief introduction of the architecture-->

## Enable the MQTT-SN Gateway

In EMQX 5.0, MQTT-SN gateways can be configured and enabled through the Dashboard, HTTP API, and configuration file `emqx.conf`. This section takes the configuration via Dashboard as an example to illustrate the operating steps. 

> On EMQX Dashboard, Click **Extensions** -> **Gateways** on the left navigation tree to open the configuration page. EMQX has already listed the gateways supported on this page, locate **MQTT-SN** and click **Setup** in the **Actions** column, then you will be directed to the **Initialize MQTT-SN** page.

Reviewer: ```suggestion
On EMQX Dashboard, click **Extensions** -> **Gateways** on the left navigation menu. On the **Gateway** page, all supported gateways lare listed. Locate **MQTT-SN** and click **Setup** in the **Actions** column. Then, you will be directed to the **Initialize MQTT-SN** page.
```

### Basic Configuration

In the **Basic Configuration** tab, you can customize your gateway ID, predefine the topic list, and set the MountPoint string for this gateway. See the texts below the screenshot for a comprehensive explanation of each field. 

![Basic Configuration](./assets/mqttsn-basic-config.png)

- **Gateway ID**: Set the unique identifier of the gateway, for example, 1. 

- **Enable Broadcast**: Set whether to allow the gateway to broadcast gateway advertisements to clients, it will broadcast the message the Gateway ID you just specified, default: **true**, optional values: **true**, **false**.

- **Enable QoS3**: Set whether to allow the gateway to support QoS 3 (Exactly-once) message delivery,  this setting is intended for basic clients that only need to send `PUBLISH` messages to the gateway; default: **true**, optional values: **true**, **false**.

- **Idle Timeout**: Set the duration (in seconds) of inactivity after which a connected MQTT-SN client will be considered disconnected. Default: **30 s**.

> - **Enable Statistics**: Set whether to allow the Gateway to collect and report statistics. default: **true**, optional values: **true**, **false**.

Reviewer: ```suggestion
- **Enable Statistics**: Set whether to allow the Gateway to collect and report statistics; default: **true**, optional values: **true**, **false**.
```
